Code/Tools/FBuild/FBuildCore/FBuildOptions.cpp

```diff
@@ -584,6 +604,7 @@ void FBuildOptions::DisplayHelp( const AString & programName ) const
             " -cacheinfo        Output cache statistics.\n"
             " -cachetrim <size> Trim the cache to the given size in MiB.\n"
             " -cacheverbose     Emit details about cache interactions.\n"
+            " -m_WorkerLimit <number_worker>\n"
```

Comment:
This documentation update seems to list the member variable name instead of the command line option name.

The style also lists all existing options in all lower-case.

Reply:
Yes, I'll correct it.